### PR TITLE
Update to Python 3

### PR DESCRIPTION
**Pull request by giovannicoppola**
Hello, I updated this great workflow to Python 3. I left the reload(sys) command, although not recommended as far as I can see on SO. I hope this helps, and thanks for creating this great Workflow.

### CODE REVIEW

**Comment by giovannicoppola** [comment]
reverted author and bundle-id fields to original

File: epoch.py [hunk 1]
```diff
@@ -1,18 +1,30 @@
-# encoding: utf-8
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Partly cloudy ⛅️  🌡️+48°F (feels +45°F, 56%) 🌬️→7mph 🌗 Sat Mar 26 10:55:59 2022
```
Comment (snooze92) [inline epoch.py:3]: This seems undesired?

File: epoch.py [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 import datetime
 import re
 import subprocess
 import sys
 import time
-from workflow import Workflow, ICON_CLOCK, ICON_NOTE
+import json
+import importlib
+
+importlib.reload(sys) #not sure this is needed.
```
Comment (snooze92) [inline epoch.py:14]: If we decided to go ahead with this change, it would be good to figure it out.

**Review by snooze92 — COMMENTED** [review]
The other Pull requests to support Python 3 seems like a smaller simpler change. I've left a minor comment on that one, but I'm hopeful we can get something good merged soon.

Let's keep this alternative approach on hold, and close without merging assuming we merge the other PR soon.

**Comment by snooze92** [comment]
Merged #10 instead.

**Pull request closed without merging**